start/1.18/template.html

```diff
@@ -31,7 +31,7 @@ <h1 id="branding-title">The Battle for Wesnoth 1.18<br/><span id="branding-sub">
 	<div id="intro">
 		<p><i>March 2024.</i> The development team is proud to present version 1.18 of <cite><a href="/">The&nbsp;Battle&nbsp;for&nbsp;Wesnoth</a></cite>!</p>
 
-		<p>This release features many new additions as well as refinements to existing content including a new Drake campaign, greatly improved graphical performance, major balance changes for multiplayer, achievements, many enhancements to the editor, and so much more! Try them out for yourself, or take a look <a href="#game">below</a> for more details.</p>
+		<p>This release features many new additions as well as refinements to existing content including a new Drake campaign, a full remake of our eldest Loyalist campaign, greatly improved graphical performance, major balance changes for multiplayer, achievements, many enhancements to the editor, and so much more! Try them out for yourself, or take a look <a href="#game">below</a> for more details.</p>
```

Comment:
I suppose they are the same, but with my particular experience with English, I would prefer 'oldest' instead of 'eldest' here.  For some reason I feel like 'eldest' is usually in the context of comparing ages of humans.

Reply:
Does 'oldest' possibly sound slightly derogatory though? I was a bit worried about that, which is why I changed it to 'eldest' - the slight personification of the campaign hopefully making it sound more like "an old friend" than "an old used car"? Gives it a connotation closer to 'venerable' than 'decrepit'?

But I'm probably over analyzing this; changed it to your suggestion.